src/Symfony/Bundle/DebugBundle/Resources/views/Profiler/dump.html.twig

```diff
@@ -45,7 +48,12 @@
 
     {% for dump in collector.getDumps('html') %}
         <div class="sf-dump sf-reset">
-            <span class="metadata">In
+            <span class="metadata">
+                {% if dump.label is defined and '' != dump.label %}
+                    <strong>{{ dump.label }}</strong> in
+                {% else %}
+                    In
```

Comment:
When there's no label we display `In FooController.php line 14` now?

Reply:
as was before, no? (see L48 before)